src/server/pfs/server/driver.go

```diff
@@ -1817,36 +1817,31 @@ func (d *driver) deleteBranchSTM(stm col.STM, branch *pfs.Branch, force bool) er
 	return nil
 }
 
-func (d *driver) scratchPrefix() string {
-	return path.Join(d.prefix, "scratch")
-}
-
 // scratchCommitPrefix returns an etcd prefix that's used to temporarily
 // store the state of a file in an open commit.  Once the commit is finished,
 // the scratch space is removed.
 func (d *driver) scratchCommitPrefix(commit *pfs.Commit) string {
-	// TODO(msteffen) this doesn't currenty (2018-2-4) use d.scratchPrefix(),
-	// but probably should? If this is changed, filepathFromEtcdPath will also
-	// need to change.
```

Comment:
slightly unrelated change - I believe this comment is outdated.  Both `scratchPrefix` and `filePathFromEtcdPath` are unused, but I believe that the "putFileRecords" collection _is_ the scratch space, which would explain it.  `filePathFromEtcdPath` appears to still be valid, but since it is unused I don't see a reason to keep it around.  @jdoliner - correct me if any of this is wrong.

Reply:
This all sounds correct to me.